Packs/XSOAR_Engineer_Training/Integrations/XSOAREngineerTraining/XSOAREngineerTraining.yml

```diff
@@ -97,6 +97,9 @@ script:
     - description: Who to send the fake email to.
       name: to
       required: true
+    - description: The body of the fake email that we don't be sending.
+      name: body
```

Comment:
Please generate a updated README.md file for your integration/script.
To do that, please run the `demisto-sdk generate-docs` command (use the -e flag for using a commad_examples file) - for more information please see: https://xsoar.pan.dev/docs/concepts/demisto-sdk#generate-docs.

Reply:
Done.